freqtrade/optimize/backtesting.py

```diff
@@ -661,7 +661,8 @@ def get_valid_price_and_stake(
                 proposed_rate=propose_rate, entry_tag=entry_tag,
                 side=direction,
             )  # default value is the open rate
-            # We can't place orders higher than current high (otherwise it'd be a stop limit entry)
+
+            # We can't place orders higher than current high (otherwise it'd be a stop limit buy)
```

Comment:
i guess that's an unintended change?

Reply:
Yeah. Looks like my messup from when merging.

Reply:
Addressed through https://github.com/freqtrade/freqtrade/pull/6692/commits/9d205132d0fb54df99a7f2b5d0a59520e07eab1b